ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -261,37 +261,63 @@ Get Model Inference
     [Documentation]    Returns the inference result after sending a POST request to a deployed
     ...    model endpoint. If token authentication is needed for the model, ${token_auth} should be
     ...    set to ${TRUE}.
-    [Arguments]    ${model_name}    ${inference_input}    ${token_auth}=${FALSE}
-    ...    ${project_title}=${NONE}
-    ${self_managed}=    Is RHODS Self-Managed
-    ${url}=    Get Model Route Via UI    ${model_name}
-    ${kserve}=    Run Keyword And Return Status    SeleniumLibrary.Page Should Contain    Single-model serving enabled
-    ${curl_cmd}=     Set Variable    curl -s ${url} -d ${inference_input}
-    IF    ${token_auth}
-        IF    "${project_title}" == "${NONE}"
-            ${project_title}=    Get Model Project    ${model_name}
+    [Arguments]    ${model_name}    ${inference_input}    ${token_auth}=${FALSE}    ${kserve_mode}=Serverless
+    ...    ${project_title}=${NONE}   ${deployment_mode}='UI'   ${service_port}=8888   ${end_point}=${NONE}
```

Comment:
You are changing the order of the arguments, this will break existing tests calling this keyword expecting the 4th argument to be `project_title`. Preserve the original order by adding your new arguments after the already existing ones

Reply:
ack moned it